src/steps/updateLocalFiles.ts

```diff
@@ -8,10 +9,7 @@ interface ExistingPackageData {
 	version?: string;
 }
 
-export async function updateLocalFiles(
-	options: Options,
-	config = { resetPackageVersion: false },
```

Comment:
Just a heads up @nowyDEV. No complaints about having the `config` in #846. Just seeing more new logic for this function now.